datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -37,16 +37,16 @@ jaydebeapi==1.2.3; python_version >= '3.0'
 jellyfish==1.0.3; python_version > '3.0'
 jpype1==1.5.0; python_version > '3.0'
 kubernetes==18.20.0; python_version < '3.0'
-kubernetes==28.1.0; python_version > '3.0'
+kubernetes==29.0.0; python_version > '3.0'
```

Comment:
We should defo have @DataDog/container-integrations confirm that this is OK.

Reply:
it's not, that's what breaks the base check because of urlib, reverting